syz-ci/syz-ci.go

```diff
@@ -144,6 +144,9 @@ type Config struct {
 	// Per-vm type JSON diffs that will be applied to every instace of the
 	// corresponding VM type.
 	PatchVMConfigs map[string]json.RawMessage `json:"patch_vm_configs"`
+	// Some commits don't live long. Cache them somewhere to increase lifetime.
```

Comment:
somewhere where?
Would be useful to say that this is a git repo URL, and we cache them there.
Also "cache" is not a well-established term, unclear what exactly it means. I would say "push", that's what we do and git users know what it means.
Also would be useful  to say what commits are pushed.
E.g. "Push all commits used in kernel builds to this git repo URL".